lib/sprinkler_web/live/temperature_live.ex

```diff
@@ -0,0 +1,15 @@
+defmodule SprinklerWeb.TemperatureLive do
```

Comment:
We can convert this to a [component](https://hexdocs.pm/phoenix_live_view/Phoenix.LiveComponent.html) when the dashboard is finished.

CC: @BelenRemedi